src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSBatchSink.java

```diff
@@ -280,6 +284,16 @@ private long extractRecordCount(Map<String, String> metadata) {
                    "This value is ignored if the bucket already exists")
     protected String location;
 
+    @Macro
+    @Description("The Content Type property is used to indicate the media type of the resource.")
+    protected String contentType;
```

Comment:
having required types in plugins is a backwards incompatible change. Please make it nullable